irohad/main/application.cpp

```diff
@@ -213,9 +214,23 @@ void Irohad::initPeerCommunicationService() {
   log_->info("[Init] => pcs");
 }
 
+void Irohad::initMstProcessor() {
+  auto mst_transport = std::make_shared<MstTransportGrpc>();
+  auto mst_completer = std::make_shared<DefaultCompleter>();
+  auto mst_storage = std::make_shared<MstStorageStateImpl>(mst_completer);
+  // TODO: @l4l magics should be fixed with options in cli branch
+  //            check #661 for details
+  auto mst_propagation = std::make_shared<GossipPropagationStrategy>(
+      wsv, std::chrono::milliseconds(5) /*emitting period*/,
```

Comment:
Every five milliseconds propagate state to five peers. I guess default preferences should be like 5 seconds and 2 peers. But this parameter should be set by the runner.

Reply:
Upon merging cli-options that should be fixed, for now it's just some random values. I can fix exact values if it matters